collect/cache/cuckooSentCache.go

```diff
@@ -285,3 +285,14 @@ func (c *cuckooSentCache) Resize(cfg config.SampleCacheConfig) error {
 	go c.monitor()
 	return nil
 }
+
+func (c *cuckooSentCache) GetMetrics() (map[string]interface{}, error) {
+	metrics := map[string]interface{}{
```

Comment:
Do we need a lock here before we get the counts?

Reply:
No, because the counts are fetched from functions that take their own locks.